Apps/Src/Temperature.c

```diff
@@ -228,7 +232,7 @@ SafetyStatus Temperature_CheckStatus(uint8_t isCharging){
 	for (int i = 0; i < NUM_MINIONS; i++) {
 		for (int j = 0; j < MAX_TEMP_SENSORS_PER_MINION_BOARD; j++) {
 			if (i * MAX_TEMP_SENSORS_PER_MINION_BOARD + j >= NUM_TEMPERATURE_SENSORS) break;
-			if (ModuleTemperatures[i][j] > temperatureLimit) {
+			if ((temperatures[i][j] > temperatureLimit) || (temperatures[i][j] == TEMP_ERR_OUT_BOUNDS)) {
```

Comment:
Do we need to have it explicitly check for TEMP_ERR_OUT_BOUNDS or is it just for readability?

Reply:
It's for readability and in case someone ever makes a code change that causes `TEMP_ERR_OUT_BOUNDS` to be a safe voltage